muon/_prot/preproc.py

```diff
@@ -193,30 +195,52 @@ def dsb(
     return toreturn
 
 
-def clr(adata: AnnData, inplace: bool = True) -> Union[None, AnnData]:
+def clr(adata: AnnData, inplace: bool = True, axis: int = 0) -> Union[None, AnnData]:
     """
     Apply the centered log ratio (CLR) transformation
     to normalize counts in adata.X.
 
     Args:
         data: AnnData object with protein expression counts.
         inplace: Whether to update adata.X inplace.
+        axis: Axis across which CLR is performed.
```

Comment:
there should probably be a check at the beginning of the function whether axis is one `0` or `1`, throwing an exception otherwise.